source/isaaclab/test/deps/isaacsim/check_ref_count.py

```diff
@@ -39,9 +39,10 @@
 import torch  # noqa: F401
 
 import isaacsim.core.utils.prims as prim_utils
-from isaacsim.core.api.simulation_context import SimulationContext
 from isaacsim.core.prims import Articulation
 
+from isaaclab.sim import SimulationCfg, SimulationContext
```

Comment:
NOTE: These are isaacsim level checks which don't use any of isaaclab. Mainly for bug repros for them. Should we revert for now?